### DISCUSSION TITLE
Number of multiple imputed datasets and percentage of incomplete cases

### DISCUSSION THREAD

**Discussion by angelrodriguez2020** [amices/mice#381] [Missing data methodology]
Dear all,

Stef van Buuren recommends that in the final run of MICE, m equals to percentage of incomplete cases. How can I calculate this percentage?

Many thanks,

Ángel Rodríguez

**Comment by stefvanbuuren** [comment]
Perhaps something like `mean(!complete.cases(nhanes))*100`?

**Reply by angelrodriguez2020** [reply]
Thank you Stef. 
As easy as that! 
Ok, my misunderstanding comes from the reading of the article: Graham JW, Olchowski AE, Gilreath TD: How many imputations are really needed? Some practical clarifications of multiple imputation theory. Prevention Science 2007, 8: 206–213. 10.1007/s11121-007-0070-9
where they propose to establish the number of imputed datasets depending on the fraction of missing
information (γ; and the tolerance to loss of power), which is more complicated to calculate than the percentage of incomplete cases.

**Reply by stefvanbuuren** [reply]
Yes, in order to calculate the fmi you need to impute first. The percentage of incomplete cases is some kind of poor-man's approximation.

**Reply by angelrodriguez2020** [reply]
Fair enough! Thanks again.
